sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="sandboxed-containers-1-3-release-notes"]
+= {sandboxed-containers-first} {sandboxed-containers-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="sandboxed-containers-1-3-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} {sandboxed-containers-version} alongside Red Hat {product-title} {product-version}.
+
+This product is fully supported and enabled by default as of {product-title} 4.10.
+
+[id="sandboxed-containers-1-3-new-features-and-enhancements"]
+== New features and enhancements
+
+
+
+[id="sandboxed-containers-1-3-bug-fixes"]
+== Bug fixes
+
+
+
+[id="sandboxed-containers-1-3-known-issues"]
+== Known issues
+
+
+
+[id="sandboxed-containers-1-3-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata is https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
```

Comment:
```suggestion
Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata are available on the https://access.redhat.com/downloads/content/290/[Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
```

Reply:
Nit: where possible, link text should use a short title that distinctly identifies the link.

https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#referring-to-web-addresses

also "errata" is a plural.

Reply:
Also, use `link:LINK_URL[LINK_TEXT]` for links. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
This was copied directly from OCP release notes. I hadn't noticed the rogue "errata is", and it looks like the OCP team has since fixed that in their release notes as well, but the link is still structured the same there... I'll fix both on my end.